lib/run_loop/device_agent/client.rb

```diff
@@ -1471,17 +1475,21 @@ def launch_cbx_runner
 
         start = Time.now
         RunLoop.log_debug("Waiting for DeviceAgent to launch...")
-        @launcher_pid = cbx_launcher.launch(options)
-
+        
         begin
-          timeout = options[:device_agent_install_timeout] * 1.5
+          retries ||= 0
+          @launcher_pid = cbx_launcher.launch(options)
+          timeout = options[:device_agent_install_timeout] * 0.3
```

Comment:
Why does we decreased this timeout?

Reply:
I want to decrease summary time of cucumber test